crates/db_schema/src/lib.rs

```diff
@@ -47,6 +47,8 @@ pub mod utils;
 #[cfg(feature = "full")]
 mod schema_setup;
 
+#[cfg(feature = "full")]
+pub use diesel_ltree::Ltree;
```

Comment:
Had to add this to get rid of dev-dependency on diesel_ltree in lemmy_api_common, which would make the ci check `lemmy_api_common_doesnt_depend_on_diesel` fail.